std/Std.hx

```diff
@@ -83,7 +83,7 @@ extern class Std {
 
 		If s is null, "null" is returned.
```

Comment:
Return type should be `Null<String>`.

Reply:
Nope, it returns "null" string, not null.

Reply:
My bad :D